management/src/clusterm/manager/api.go

```diff
@@ -72,11 +73,12 @@ func (m *Manager) apiLoop(errCh chan error) {
 	}
 }
 
-type postCallback func(tagsOrAddrs []string, sanitizedExtraVars string) error
+type postCallback func(tagsOrAddrs []string, flags ActionFlags) error
```

Comment:
I think we should just add a third argument here for now or replace the `tagsOrAddrs` with `*APIRequest`. Going forward once #99 is fixed I was planning to actually make this callback just take `*APIRequest` as `extraVars` will be part of the body as well.